src/schematic-components/SchematicElement.tsx

```diff
@@ -73,5 +76,21 @@ export const SchematicElement = ({
     return <RenderError text={element.message} />
   }
 
+  if (element.ftype === "simple_bug") {
+    const obj: SchematicTextType = {
+      type: "schematic_text",
+      schematic_component_id: "schematic_component_simple_bug_0",
+      schematic_text_id: "schematic_text_0",
+      anchor: "top",
+      position: {
+        x: -0.5,
+        y: 1,
+      },
+      text: element.name,
+      rotation: 0,
+    }
+    return <SchematicText schematic_text={obj} />
```

Comment:
hmm i don't get how this works, there is logic on line 25 that should make it so this is never reached

```tsx
if (element.type === "schematic_component") {
    return (
      <SchematicComponent
        component={collectElementRefs(element, allElements) as any}
      />
    )
  }
```

did you test this? If so what story shows that it works?

Reply:
I looked at the PR preview and it didn't work.

https://schematic-viewer-git-add-bug-name-tscircuit.vercel.app/?path=/story/ledcircuitbuilder--led-circuit-builder